parlai/core/torch_agent.py

```diff
@@ -1368,9 +1369,11 @@ def _set_text_vec(self, obs, history, truncate):
         # check truncation
         if obs.get('text_vec') is not None:
             truncate_left = not self.history_reversed
+            text_length = len(obs['text_vec'])
             truncated_vec = self._check_truncate(
                 obs['text_vec'], truncate, truncate_left
             )
+            obs.force_set('if_text_truncate', text_length != len(truncated_vec))
```

Comment:
is != right here? hm

Reply:
Why is it not? hmmm

Reply:
I don't get it. Should it be `> ` instead or != doesn't work this way in python like in cpp?